include/grpc++/impl/codegen/async_stream.h

```diff
@@ -52,11 +52,14 @@ class ClientAsyncStreamingInterface {
 
   /// Request notification of the reading of the initial metadata. Completion
   /// will be notified by \a tag on the associated completion queue.
+  /// This call is optional, but if it is used, it cannot be used concurrently
+  /// with Read
```

Comment:
this is a nit/pet-peeve of mine (but that it's also done inconsistently in this PR): Can I haz periods at end of sentences? :grimacing: